examples/helloworld-cdi2-se/README.MD

```diff
@@ -31,4 +31,4 @@ Run the example as follows:
 
```

Comment:
Copyright years in headers shall be adjusted in all modified files

Reply:
Done with latest commit pushed just now.